recipes_source/regional_aot.py

```diff
@@ -0,0 +1,238 @@
+
```

Comment:
we also need to add a carditem and to the toctree in the https://github.com/pytorch/tutorials/blob/main/recipes_index.rst.